Dockerfile_build

```diff
@@ -2,7 +2,69 @@
 
 FROM golang:1.12
 
-RUN apt-get update && apt-get install -y --no-install-recommends \
-		ruby \
-		ragel=6.9-1.1+b1 \
-	&& rm -rf /var/lib/apt/lists/*
+# Install common packages
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y \
+    ruby \
+    ca-certificates curl file \
+    build-essential \
+    autoconf automake autotools-dev libtool xutils-dev && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install ragel
+ENV RAGEL_VERSION=6.10
```

Comment:
Yeah, so now we  can `brew install ragel`, right?

Reply:
That is the intent, yes.